components/button/index.css

```diff
@@ -13,76 +13,109 @@ governing permissions and limitations under the License.
 @import "../vars/css/scales/spectrum-large.css";
 @import "../vars/css/scales/spectrum-medium.css";
 @import "../commons/basebutton.css";
```

Comment:
We'll need to remove these lines, as we're no longer using the `vars` package.

Reply:
removed